src/napari_plugin_manager/base_qt_package_installer.py

```diff
@@ -534,8 +611,8 @@ def _log(self, msg: str):
             self._output_widget.append(msg)
 
     def _get_tool(self, tool: InstallerTools):
```

Comment:
Return type annotation

Reply:
Added a bunch more in https://github.com/napari/napari-plugin-manager/pull/177, @Czaki